social-compose/src/main/java/com/amity/socialcloud/uikit/community/compose/UserMentionBehavior.kt

```diff
@@ -0,0 +1,12 @@
+package com.amity.socialcloud.uikit.community.compose
+
+import android.content.Context
+
+open class NoomUserMentionBehavior {
```

Comment:
Using `Noom` as a prefix to separate our changes from the Amity ones (they're using `Amity` as a prefix).